### PR TITLE
NOISSUE - Update authors list

### PR DESCRIPTION
**Pull request by drasko**
Add Dejan's recognition

Signed-off-by: drasko <drasko.draskovic@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,6 @@ Additional practical information, news and tutorials can be found on the [Mainfl
 ## Authors
 Main architect and BDFL of Mainflux project is [@drasko][drasko].
 
-
 Additionally, [@nmarcetic][nikola] and [@janko-isidorovic][janko] assured
 overall architecture and design, while [@manuIO][manu] and [@darkodraskovic][darko]
```
Comment (manuio) [inline README.md]: `[@manuio][manu]`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (drasko) [reply README.md]: So?

Comment (manuio) [reply README.md]: IO io

**Review by manuio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (drasko) [reply README.md]: Fixed

**Review by manuio — APPROVED** [review]
LGTM

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Pull request merged**